controllers/triggerinstance/triggerinstance_controller.go

```diff
@@ -146,26 +146,26 @@ func (r *Reconciler) createDefaultInstance(ctx context.Context) {
 	}
 	if util.IgnoreNotFound(err) != nil {
 		logger.Errorf("getting default instance failed, the default TriggerInstance may not be created: %s", err)
-		return
-	}
 
```

Comment:
This logic doesn't look right to me. You are creating the default instance only if there is an error (and the error is NOT `not found`).

However, we want to create the default instance ONLY when the error is `not found`.

Reply:
Okay, I got it wrong.